index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
@tysongach Your choice, of course, but I'd have thought that you wouldn't want (even arguable) misspellings on an otherwise well-designed and well-written project website.